src/contribute/release-guide.md

```diff
@@ -511,7 +512,220 @@ If there are no issues, reply on the vote thread to close the voting. Then, tall
     There are no disapproving votes.
 
     Thanks everyone!
+    
+### Run validation tests
+All tests listed in this [spreadsheet](https://docs.google.com/spreadsheets/d/1qk-N5vjXvbcEk68GjbkSZTR8AGqyNUM-oLFo_ZXBpJw/edit#gid=152451807)
```

Comment:
Change to https://s.apache.org/beam-release-validation ?

Reply:
Fixed in https://github.com/apache/beam-site/pull/468/commits/ce2286c3569959ea1f3821bf8522127ed7c73bc4 .